engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/gateway/InclusiveGatewayTest.java

```diff
@@ -127,22 +132,6 @@ public void shouldSplitOnInclusiveGateway() {
         .extracting(v -> tuple(v.getProcessInstanceKey(), v.getElementId()))
         .contains(tuple(processInstance3, "end3"));
 
```

Comment:
Sorry I forgot this one:

🔧 We'll also need to change this one to: if no conditions are fulfilled, then the default flow is taken

Reply:
It's done.